### DISCUSSION TITLE
[Feature] "Upload to Immich" mobile action

### DISCUSSION THREAD

**Discussion by ralienpp** [immich-app/immich#10782] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

My use case:
- regularly add new photos from several mobiles to an album shared with extended family (parents share photos of grandchildren with grandparents, aunts et al.)
- we take many pictures with our phones, but only a small subset ends up in this album
- the dissemination of these photos is the main and only reason we have Immich

The current experience is that Immich on mobile assumes that all the photos will be uploaded to the server, and once there - you can choose to add some of them to an album. However, we prefer to cherry-pick them from the phone's gallery and say "Upload to Immich". A good alternative would be to browse the phone's local gallery via Immich, click a pic and press "Add to shared album", which under the hood does "[Upload to Immich] and add to shared album".

The motivation to not take the "just upload everything" route is that Immich is running on a rather modest system and I'd prefer to not stuff it with unnecessary images.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by mmomjian** [comment]
Thanks for this request! Dup #2659
